app/build.gradle

```diff
@@ -110,6 +110,13 @@ dependencies {
     implementation 'com.squareup.retrofit2:retrofit:2.7.1'
     implementation "androidx.room:room-rxjava2:$room_version"
     testImplementation "androidx.arch.core:core-testing:2.1.0"
+
+    // Pref
+    def preference_version = "1.1.0"
```

Comment:
move this into `gradle.properties` and follow their style

Reply:
Done!